website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration-with-github.md

```diff
@@ -41,6 +41,6 @@ dbt test --models state:modified+
 Because dbt Cloud manages deferral and state environment variables, there is no need to specify `--defer` or `--state` flags. **Note:** Both jobs need to be running dbt v0.18.0 or newer.
 
 <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/ci-deferral.png" title="Jobs that run
-on pull requests may select another job from the same project for deferral and comparison"/>
+on pull requests may select "self" or another job from the same project for deferral and comparison"/>
```

Comment:
The quotes here are breaking the build, since the website thinks the `"sel..` quote is the closing quote for the `title="Jobs...`.

Try this instead — the `&quot;` string gets interpreted as a quote symbol
```suggestion
on pull requests may select &quot;self&quot; or another job from the same project for deferral and comparison"/>
```

Heads up — before submitting PRs, you can run `make build` from your command line (from the `website directory`) — errors like these will be picked up locally, which means you can diagnose them and fix them faster! If you can't figure out what's causing the error, open the PR as a draft PR, and then assign one of us as reviewer to get some help!